### ISSUE TITLE
About loss hair and loss face  different from paper 

### ISSUE THREAD

**Issue by KIM7AZEN** [oneThousand1000/HairMapper#22]
in parer   loss hair is difine as 
 
<img width="291" alt="image" src="https://github.com/oneThousand1000/HairMapper/assets/19183626/b6f96bf1-af41-4a1b-82d9-9aed9ece2dd1">

 but in train_mapper.py  this loss do not dot with hair mask..  wondering why

        loss_l2_latent = self.latent_l2_loss(w_hat, res_w)
        loss_dict['loss_l2_latent'] = float(loss_l2_latent)
        loss += loss_l2_latent * self.latent_l2_lambda

        loss_l2_img = torch.mean(((res_x - x_hat)) ** 2, dim=[0, 1, 2, 3])  ### do not dot with hair mask
        loss_dict['loss_l2_res_img'] = float(loss_l2_img)
        loss += loss_l2_img * self.img_l2_lambda_res

        loss_l2_img = torch.mean(((origin_img - x_hat) * mask) ** 2, dim=[0, 1, 2, 3])
        loss_dict['loss_l2_origin_img'] = float(loss_l2_img)
        loss += loss_l2_img * self.img_l2_lambda_origin
